### PR TITLE
Feature/zip removed

### PR DESCRIPTION
**Pull request by peggles2**
This pull request populates the zip code search field with the new zip location when the map is dragged.

### CODE REVIEW

File: src/client/components/organisms/office-map/office-map.jsx [hunk 1]
```diff
@@ -336,8 +342,7 @@ class OfficeMapApp extends React.PureComponent {
     // pass array of points with lats and lngs
 
     const { points, map } = this.state
-    const { onFieldChange, selectedItem, newCenter, onDragEnd, hoveredMarkerId, total } = this.props
-
+    const { onFieldChange, selectedItem, newCenter, onDragEnd, hoveredMarkerId, total, zip } = this.props
```
Comment (kscoulter) [inline src/client/components/organisms/office-map/office-map.jsx:345]: you don't need to destructure zip here, doesn't seem like you're using that variable anymore

File: src/client/components/organisms/office-map/office-map.jsx [hunk 2]
```diff
@@ -139,7 +139,13 @@ class OfficeMapApp extends React.PureComponent {
   }
 
   componentWillReceiveProps(nextProps) {
-    const { items, shouldCenterMap } = nextProps
+    const { items, shouldCenterMap, zip } = nextProps
+    if (zip !== this.props.zip) {
```
Comment (kscoulter) [inline src/client/components/organisms/office-map/office-map.jsx:143]: what happens on other pages that use office map but don't pass the zip?

**Review by kscoulter — CHANGES_REQUESTED** [review]
You should probably add the zip to prop types in office map component.

File: src/client/components/organisms/office-map/office-map.jsx (hunk 2 above)
Comment (kscoulter) [reply src/client/components/organisms/office-map/office-map.jsx:143]: @peggles2, did you see this question?

Comment (peggles2) [reply src/client/components/organisms/office-map/office-map.jsx:143]: only other office page is find a lender and if you look at the code you see I updated it to handle that.

**Review by kscoulter — APPROVED** [review]

**Pull request merged**